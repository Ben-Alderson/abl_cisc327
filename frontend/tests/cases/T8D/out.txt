error(You must login to create accounts)


