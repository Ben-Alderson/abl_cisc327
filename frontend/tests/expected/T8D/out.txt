Enter command: Enter login: Enter command: Enter command: Enter the new account's number: Enter the new account's name: ERROR: You need to login first
Enter command: 